src/components/KAlert/KAlert.cy.ts

```diff
@@ -85,21 +86,25 @@ describe('KAlert', () => {
         message,
       },
       slots: {
-        default: `<span data-testid="default-slot-content">${defaultSlotContent}</span>`,
+        default: h('span', {}, defaultSlotContent),
       },
     })
 
     cy.get('.alert-message').should('be.visible').and('have.text', defaultSlotContent)
   })
 
   it('displays icon passed through icon slot', () => {
+    const dragon = '🐉'
```

Comment:
can we just check for a string here instead of an emoji?

Reply:
I can change this, but for the record the test is completely functional as is and this shouldn't be considered blocking feedback ☹️

Reply:
We shouldn't be using emojis in tests 🤷🏼